bench-vortex/scripts/public_pi_run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# List files in the current directory
+dir=$(dirname ${BASH_SOURCE[0]})
+files=$(ls $dir/../public_bi/benchmark)
+
+for file in $files; do
+    echo "Running public BI: $file"
+      
+    file_lowercase=$(echo "$file" | tr '[:upper:]' '[:lower:]')
```

Comment:
Why is the conversion to lowercase necessary if this is how the file is originally named?

Reply:
Its no how the file is named, the files are named inconsistently

Reply:
Is this how they are provided by the benchmark, or did we name them? In case of the latter I'd prefer if we just lowercase the actual filename. Agreed on displaying filenames consistently is nice but I think it's not optimal that the names then no longer corresponding to the actual file names.

Reply:
We cannot control the names, just the enums